fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformService.java

```diff
@@ -55,6 +55,9 @@ CommandProcessingResult makeLoanRepaymentWithChargeRefundChargeType(LoanTransact
     @Transactional
     CommandProcessingResult makeInterestPaymentWaiver(JsonCommand command);
 
+    @Transactional
```

Comment:
We dont need command handler or API for this functionality. The only incoming parameter will be a Loan entity